package.json

```diff
@@ -71,7 +71,7 @@
     "@commitlint/config-conventional": "^17.8.1",
     "@emotion/cache": "^11.11.0",
     "@emotion/react": "^11.11.4",
-    "@instructure/ui-scripts": "8",
+    "@instructure/ui-scripts": "9",
```

Comment:
is this dependency even needed?

Reply:
probably, or before we "unified" lots of our scripts. other question: does `command-utils` need to be a public package?

Reply:
I've removed this dependency from here.

> probably, or before we "unified" lots of our scripts. other question: does command-utils need to be a public package?

nope, but I'll need to double check

Reply:
IMO we should set everything thats possible to private in a future pr